source/rover.js

```diff
@@ -17,6 +17,15 @@ this.initialize = function() {
     if ( this.id === "source/rover.vwf" ) {
         return;
     }
+
+    //TODO: Set this properly (we shouldn't have to set this explicity in here...
+    // this.transform gets set automatically for ONE of the rovers, but not the other one)
+    this.transform = [
+        1,  0,  0,  0,
+        0,  1,  0,  0,
+        0,  0,  1,  0,
+        0,  0,  0,  1
+    ];
```

Comment:
Ok, it looks like the transform property is undefined until a translation, rotation, or transform is set (after initialization). This should probably not be the case as I'm sure the threejs object has a transform. I checked node3.vwf and there is no getter for the transform property. The others us an identity transform if the property has no value. Is there any reason transform should not do the same? @eric79 @davideaster